ipn/ipnserver/actor.go

```diff
@@ -237,6 +238,11 @@ func connIsLocalAdmin(logf logger.Logf, ci *ipnauth.ConnIdentity, operatorUID st
 		// Linux.
 		fallthrough
 	case "linux":
+		if !buildfeatures.HasUnixSocketIdentity {
+			// Everybody is an admin if support for unix socket identities
+			// is omitted for the build.
+			return true
+		}
```

Comment:
This does feel like a bit of a footgun, though unlikely that someone trips over it; we can't easily say "only root" since we don't have any peer creds, but maybe worth a `sync.Once`'d log message or something?